src/modules/browse/components/BrowseInfo/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+
+class BrowseInfo extends React.Component {
+  render() {
+    const { datastore } = this.props;
+
+    if (datastore.uid === 'databases' || datastore.uid === 'journals') {
```

Comment:
Long-term we'll probably want to be able to ask the datastore whether it's browseable, and/or wrap it in a presenter so that this could just return `datastorePresenter.browseLink();`

Reply:
I moved this browsable check into a the Pride module so that it can be updated easier later if we choose.